.github/workflows/build.yml

```
@@ -425,7 +425,7 @@ jobs:
           get_status_emoji() {
             local size=$1
             local diff=$2
-            if [ $size -gt 2400 ]; then
+            if [ $size -gt 3200 ]; then
```

This can be unpredictable because of the compressed sizes. Lets be conservative and aim to never reach the 3Mb runtime size.
```suggestion
            if [ $size -gt 3000 ]; then
```